media-proxy/include/mesh/st2110.h

```diff
@@ -0,0 +1,57 @@
+#ifndef ST2110_H
+#define ST2110_H
+
+#include <thread>
+#include <bsd/string.h>
+#include <arpa/inet.h>
+#include <mtl/st_pipeline_api.h>
+#include <mtl/st30_pipeline_api.h>
+
+#include "conn.h"
+#include "mesh_dp.h"
+#include "logger.h"
+
+namespace mesh::connection {
+
+#define ST_APP_PAYLOAD_TYPE_ST30 (111)
+#define ST_APP_PAYLOAD_TYPE_ST20 (112)
+#define ST_APP_PAYLOAD_TYPE_ST22 (114)
+
+/**
+ * ST2110
+ *
+ * Base abstract class of ST2110. ST2110Rx/ST2110Tx
+ * inherit this class.
+ */
+class ST2110 : public Connection {
+  public:
+    static st_frame_fmt mesh_video_format_to_st_format(int fmt);
+    static st30_fmt mesh_audio_format_to_st_format(int fmt);
+    static st30_sampling mesh_audio_sampling_to_st_sampling(int sampling);
+    static st30_ptime mesh_audio_ptime_to_st_ptime(int ptime);
```

Comment:
Consider removing the prefix `mesh_`. It is redundant since the static methods are defined under `namespace mesh::connection`.